<ui version="4.0">
 <class>PanelEstudiante</class>
 <widget class="QMainWindow" name="PanelEstudiante">
    <property name="geometry">
     <rect><x>0</x><y>0</y><width>500</width><height>400</height></rect>
    </property>
    <property name="windowTitle">
     <string>MenULE - Panel Estudiante</string>
    </property>
    <property name="windowState">
     <set>Qt::WindowMaximized</set>
    </property>
    <property name="styleSheet">
     <string>
QMainWindow {
        background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                                                            stop:0 #1565c0, stop:1 #90caf9);
}
QLabel {
        font-size: 20px;
        color: white;
        font-weight: bold;
}
QPushButton {
        background-color: #1976d2;
        color: white;
        font-weight: bold;
        border-radius: 8px;
        padding: 6px;
        min-width: 200px;
        max-width: 200px;
}
QPushButton:hover {
        background-color: #1565c0;
}
     </string>
    </property>
    <widget class="QWidget" name="centralwidget">
     <layout class="QVBoxLayout" name="mainLayout">
        <item>
         <spacer name="verticalSpacerTop">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelTitulo">
            <property name="text">
             <string>Bienvenido/a</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter</set>
            </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacerBetweenTitleAndButtons">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnVerMenu">
            <property name="text">
             <string>Ver Menú</string>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnConfiguracion">
            <property name="text">
             <string>Configuración</string>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnHistorialReservas">
            <property name="text">
             <string>Historial de Reservas</string>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnReportarIncidencia">
            <property name="text">
             <string>Reportar Incidencia</string>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDarseDeBaja">
            <property name="text">
             <string>Darse de baja</string>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacerBottom">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
         </spacer>
        </item>
     </layout>
    </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
